models/cm_default.nestml

```diff
@@ -0,0 +1,185 @@
+"""
+Example compartmental model for NESTML
+
+Description
++++++++++++
+Corresponds to standard compartmental model implemented in NEST.
+
+References
+++++++++++
+
+
+See also
+++++++++
+
+
+Author
+++++++
+Willem Wybo
+"""
+neuron cm_default:
+
+    state:
+
+        # the presence of the state variable [v_comp]
```

Comment:
Hey the description was indeed outdated but as far as I see the state must still be defined so that the parser does not complain that the variable in the equations can't be resolved.